x-pack/plugins/serverless_search/public/application/components/connectors_ingestion.tsx

```diff
@@ -50,7 +50,10 @@ export const ConnectorIngestionPanel: React.FC<{ assetBasePath: string }> = ({ a
       </EuiFlexItem>
       <EuiFlexGroup direction="row" justifyContent="flexStart" alignItems="center">
         <EuiFlexItem grow={false}>
-          <EuiLink onClick={() => createConnector()}>
+          <EuiLink
+            data-test-subj="serverlessSearchConnectorIngestionPanelSetUpAConnectorLink"
```

Comment:
nit: why that change in that PR?

Reply:
This was an auto commit. Probably was already fixed by some other PR, I'll see if I get any conflicts when I merge